src/Symfony/Component/DependencyInjection/Loader/Configurator/ServicesConfigurator.php

```diff
@@ -137,4 +139,9 @@ public function __construct(ContainerBuilder $container, PhpFileLoader $loader,
     {
         return $this->set($id, $class);
     }
+
+    final public function getPath()
```

Comment:
can we get rid of the method? It'd be better to find another way to communicate the path to the outside and not pollute autocompletion (since this is what this configurator is about). Maybe passing as constructor arguments?

Reply:
Hope this is what you had in mind